dom.bs

```diff
@@ -4175,57 +4157,80 @@ returns as mask:
  <li><dfn const for=Node>DOCUMENT_POSITION_IMPLEMENTATION_SPECIFIC</dfn> (32, 20 in hexadecimal).
 </ul>
 
-The <dfn method for=Node>compareDocumentPosition(<var>other</var>)</dfn>
-method must run these steps:
+<p>The <dfn method for=Node><code>compareDocumentPosition(<var>other</var>)</code></dfn> method,
```

Comment:
Here or later, wrap `contains(other)` below in `<code>` as well so that it's also orange?